src/app/main/home/screen.test.tsx

```diff
@@ -0,0 +1,73 @@
+import { NavigationContainer } from '@react-navigation/native';
+import { createStackNavigator } from '@react-navigation/stack';
+import { apiService } from '@shared/api';
+import { userService } from '@shared/user';
+import { fireEvent, render, RenderAPI, waitFor } from '@testing-library/react-native';
+import { userPaginationResponse } from '@tests/fixtures';
+import { scrollDownEventData, TestRootComponent } from '@tests/helpers';
+import React from 'react';
+import { act, ReactTestInstance } from 'react-test-renderer';
+import { of } from 'rxjs';
+import { HomeScreen } from './screen';
+
+describe('Home screen', () => {
+  let component: RenderAPI;
+  let usersList: ReactTestInstance;
+  const searchUsersSpy = jest.spyOn(userService, 'search');
+
+  function initComponent(): RenderAPI {
+    const { Screen, Navigator } = createStackNavigator();
+
+    return render(
+      <TestRootComponent>
+        <NavigationContainer>
+          <Navigator>
+            <Screen name='Home' component={HomeScreen} />
+          </Navigator>
+        </NavigationContainer>
+      </TestRootComponent>
+    );
+  }
+
+  beforeAll(() => {
+    jest.spyOn(apiService, 'get').mockImplementation((endpoint) => {
+      if (endpoint === '/users') {
+        return of(userPaginationResponse) as any;
```

Comment:
can w avoid using `as any` here?

Reply:
No, but I replaced it with `Observable<any>` according to [apiService.request](https://github.com/RonasIT/react-native-starter/blob/936b1e659f558cd92862eb6bc6b48f7b66c15aa5/src/app/shared/api/service.ts#L56) which is [used](https://github.com/RonasIT/react-native-starter/blob/936b1e659f558cd92862eb6bc6b48f7b66c15aa5/src/app/shared/api/service.ts#L29) for `apiService.get`